deliver/lib/deliver/upload_metadata.rb

```diff
@@ -250,7 +250,7 @@ def set_trade_representative_contact_information(v, options)
       UI.user_error!("`trade_representative_contact_information` must be a hash", show_github_issues: true) unless info.kind_of?(Hash)
 
       TRADE_REPRESENTATIVE_CONTACT_INFORMATION_VALUES.each do |key, option_name|
-        v.send("#{key}=", info[option_name].chomp) if info[option_name].chomp
+        v.send("#{key}=", info[option_name]) if info[option_name]
```

Comment:
I think we still want to call `chomp` to strip newlines if `option_name` is present, so this should be:

```ruby
v.send("#{key}=", info[option_name].chomp)) if info[option_name]
```